.gitignore

```diff
@@ -0,0 +1 @@
+.bitrise*
```

Comment:
please include _tmp as well if you use that dir for testing in case if someone forgets to delete before push